src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -53,7 +53,7 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setPhone(ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get()));
         }
         if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
-            editPersonDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get()));
+            editPersonDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).orElse("")));
```

Comment:
Similar to my review on your Address PR #74 -- I'm unsure whether `orElse` is required